common/changes/@microsoft/rush/patch-1_2020-04-11-10-11.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "",
```

Comment:
```suggestion
      "comment": "Fix an issue where, on some minimal systems, Rush used a missing shell command to detect an application path.",
```

Reply:
Thanks! I didn't find a CONTRIBUTION.md. I didn't know what should todo.